packages/replay/src/main.ts

```diff
@@ -348,7 +348,7 @@ async function doUploadRecording(
   );
   maybeLog(verbose, `Created remote recording ${recordingId}, uploading...`);
   if (metadata) {
-    maybeLog(verbose, `Setting recording metadata for ${recordingId}`);
+    maybeLog(verbose, `Setting recording metadata for ${recordingId}: ${metadata}`);
```

Comment:
just want to clarify that metadata is a string

Reply:
Ya know what, this should probably have a JSON stringify in there. I'll submit a patch on top